testfixtures/logcapture.py

```diff
@@ -60,7 +66,12 @@ def __init__(self, names=None, install=True, level=1, propagate=None,
         self.attributes = attributes
         self.recursive_check = recursive_check
         self.old = defaultdict(dict)
-        self.clear()
+        #: The log level above which checks must be made for logged events.
+        if ensure_checks_above:
+            self.ensure_checks_above = ensure_checks_above
+        else:
+            self.ensure_checks_above = self.__class__.default_ensure_checks_above
```

Comment:
Pretty sure I suggested this already, but the above is really horrible Python...
(also, the `is None` is important, otherwise `LogCapture(ensure_checks_above=0)` will not work)
```suggestion
        if ensure_checks_above is None:
            self.ensure_checks_above = self.default_ensure_checks_above
        else:
            self.ensure_checks_above = ensure_checks_above
```